==> Preparing data..
Files already downloaded and verified
Files already downloaded and verified
scratch prediction  10.26
Training the network or loading the network
Traceback (most recent call last):
  File "main.py", line 159, in <module>
    net.load_state_dict(torch.load(args.load_net))
  File "/home/juanma/anaconda3/envs/dbviz_env/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1227, in load_state_dict
    raise RuntimeError('Error(s) in loading state_dict for {}:\n\t{}'.format(
RuntimeError: Error(s) in loading state_dict for ResNet:
	While copying the parameter named "conv1.weight", whose dimensions in the model are torch.Size([64, 3, 3, 3]) and whose dimensions in the checkpoint are torch.Size([64, 3, 7, 7]), an exception occurred : ('The size of tensor a (3) must match the size of tensor b (7) at non-singleton dimension 3',).